### PR TITLE
Adding a develop menu option to open Remix IDE

### PR DESCRIPTION
**Pull request by evertonfraga**
I'd suggest we use a shorter name on `<title>` tag, so it displays nicer on Mist interface, and a more straightforward icon would help as well :)  

@yann300 @chriseth I could raise a PR on remix repo for that, just tell me the directions (image file and new title)

image

image

### CODE REVIEW

File: modules/preloader/mistUI.js [hunk 1]
```diff
@@ -131,6 +131,17 @@ ipcRenderer.on('uiAction_runTests', (e, type) => {
     }
 });
 
+// Open Browser Solidity
+ipcRenderer.on('uiAction_openBrowserSolidity', () => {
+    Tabs.upsert('browserSol', {
+        position: Tabs.find().count(),
+        name: 'Remix',
+        redirect: 'https://ethereum.github.io/browser-solidity/',
```
Comment (chriseth) [inline modules/preloader/mistUI.js]: Please use `https://remix.ethereum.org`

File: modules/menuItems.js [hunk 2]
```diff
@@ -357,9 +358,17 @@ let menuTempl = function (webviews) {
                 log = 'Couldn\'t load log file.';
             }
         },
-    },
-    ];
+    }];
 
+    if (Settings.uiMode === 'mist') {
+        devToolsMenu.push({
+            label: i18n.t('mist.applicationMenu.develop.browsersol'),
```
Comment (chriseth) [inline modules/menuItems.js]: I think it is better to use `remix` throughout instead of `browsersol` or `BrowserSolidity` (below).

**Comment by chriseth** [comment]
@evertonfraga good idea! Please change the title to just "Remix" in github.com/ethereum/browser-solidity. We could also use "Remix - Solidity Realtime...".
What do you mean by "more straightforward icon"?

**Comment by frozeman** [comment]
Ok please rename to "Open Remix IDE" on the menu.

Then please open in a `windows.createPopupWindow`
This should then have web3 and all accounts by default.

**Comment by yann300** [comment]
cool! I sent you logos in the gitter channel that you could use.

**Comment by evertonfraga** [comment]
- [x] Change url to `https://remix.ethereum.org`
- [x] Open Remix in a popup, with access to all accounts
- [x] Use `Remix` naming instead of `browserSol` throughout code

Opening (the recoding is cropped, Remix opens centered): 

remix mov


Accounts: 

remix-2 mov

**Comment by chriseth** [comment]
That looks nice! Is it obvious to the user that this remix instance is tied to mist?

**Comment by frozeman** [comment]
Yes it is as he opens it from the Mist menu. Tho we have to improve the visibility of the current running node (type) and maybe also improve that in Remix. E.g. @chriseth @yann300 

For example highlighting the current provider used (JS, web3, etc) somewhere  always visible in the Remix interface.

**Comment by frozeman** [comment]
@evertonfraga it would be good if we set the title of the popup to "Remix IDE"

**Comment by frozeman** [comment]
@chriseth i would also prefer a dark theme for the Remix editor. this would immediately make the whole interface cleaner looking as the editor pane will be clearly separated from the debug panel

**Comment by evertonfraga** [comment]
@frozenan There'll be a PR for Remix repo, where i'll change this title,
and try to fix the `window.prompt` compatibility issue.

**Comment by frozeman** [comment]
@evertonfraga please push another empty commit, something is wrong with travis

**Pull request merged**